src/lib389/lib389/_mapped_object.py

```diff
@@ -148,6 +148,19 @@ def exists(self):
 
         return True
 
+    def search(self, scope="subtree", filter='objectclass=*'):
```

Comment:
We already have this functionality in DSLDapObjects.filter(). I think it can be safely used here.

DSLdapObject (singular instance) is for single entry operations so it makes less sense to add 'search' function here.